### PR TITLE
Feature/mhip v2

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

_Description of the feature or fix with link to the issue if existing_

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevant
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# top-most EditorConfig file
```
Comment (bionara) [inline .editorconfig:1]: May want to git ignore this?

Comment (aikewoody) [reply .editorconfig:1]: Interesting discussion, I would say this is nice to have in the codebase. All the settings in the config match our ESLint and Prettier settings. @bionara Do you have an argument in favour of git ignoring it?

Comment (bionara) [reply .editorconfig:1]: Not as strong as your argument for not ignoring it :)

**Review by bionara — APPROVED** [review]
Looks good. lots of linting! nice work, Aike!

**Comment by selvach** [comment]
Please hold of merging until we chat!

**Comment by selvach** [comment]
Merge when ready!

**Review by selvach — APPROVED** [review]

**Pull request merged**